docs/kubernetes/goblin-all-in-one.yaml

```diff
@@ -0,0 +1,264 @@
+kind: ConfigMap
```

Comment:
BTW this whole file/folder is still missing a guide, but I will add this later because I'd rather merge the PR and iron out potential bugfixes before we get drown in merge conflicts.